src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspaceManager.cs

```diff
@@ -26,7 +26,7 @@ internal class RemoteWorkspaceManager
         MefHostServices.DefaultAssemblies
             .Add(typeof(AspNetCoreEmbeddedLanguageClassifier).Assembly)
             .Add(typeof(BrokeredServiceBase).Assembly)
-            .Add(typeof(RazorAnalyzerAssemblyResolver).Assembly)
+            .Add(typeof(IRazorLanguageServerTarget).Assembly)
```

Comment:
Why has this changed?

Reply:
Because it targets both .NET and netstandard. In the netstandard version `RazorAnalyzerAssemblyResolver` no longer exists. It just seemed neater to change the type we're referencing instead of #if-defing the assembly add.